extensions/plugin-history-sync/src/historySyncPlugin.tsx

```diff
@@ -103,18 +103,14 @@ export function historySyncPlugin<
         const activityNames = Object.keys(options.routes);
 
         if (path) {
-          for (let i = 0; i < activityNames.length; i += 1) {
-            const activityName = activityNames[i] as K;
-            const routes = normalizeRoute(options.routes[activityName]);
+          for (const activityName of activityNames) {
+            const routes = normalizeRoute(options.routes[activityName as K]);
 
-            for (let j = 0; j < routes.length; j += 1) {
-              const route = routes[j];
-
-              const template = makeTemplate(route, options.urlPatternOptions);
+            for (const route of routes) {
+              const template = makeTemplate(route);
```

Comment:
```suggestion
              const template = makeTemplate(route, options.urlPatternOptions);
```

rebase 시점에 새로운 파라미터가 추가 됐어요. 반영 부탁드릴게요~